### DISCUSSION TITLE
Integrating with Tauri Apps

### DISCUSSION THREAD

**Discussion by mehedimi** [SeaQL/sea-orm#1628] [Ideas]
I think you guys can add an example of how Sea-ORM works inside the Tauri command of Tauri Apps.

Thanks!

**Comment by billy1624** [comment]
Hey @mehedimi, there are example apps on this list https://github.com/SeaQL/sea-orm/blob/master/COMMUNITY.md
For examples
- https://github.com/Deaftone/Deaftone
- https://github.com/tsirysndr/music-player
